opds.py

```diff
@@ -848,6 +848,9 @@ def _create_entry(self, work, license_pool, edition, identifier,
 
     def _make_entry_xml(self, work, license_pool, edition, identifier):
 
+        if work and not edition:
+            edition = work.presentation_edition
+
```

Comment:
I think this only happens on a few titles. This is indicative of a failure elsewhere in the system, and this doesn't fix that failure. So if there's a downside to this change, it's that it pushes the point of eventual failure further away. But for the metadata wrangler it seems to push the point of failure so far away that it won't be encountered with our current functionality.

For the record, the work on the metadata wrangler that appears to have this problem is https://unglue.it/api/id/work/130412/, "It's Complicated" by danah boyd.